shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/ProjectionsTokenGeneratorTest.java

```diff
@@ -59,27 +59,25 @@
 
     private static final String TEST_DERIVED_PROJECTION_ALIAS = "TEST_DERIVED_PROJECTION_ALIAS";
 
-    private static final int TEST_STOP_INDEX = 2;
-
     private static final String TEST_LOGIC_TABLE_NAME = "TEST_LOGIC_TABLE_NAME";
 
-    private static final String TEST_ACTUAL_TABLE_NAME = "TEST_ACTUAL_TABLE_NAME";
-
     private static final String TEST_ACTUAL_TABLE_NAME_WRAPPED = "TEST_ACTUAL_TABLE_NAME_WRAPPED";
 
     private static final String TEST_OTHER_DERIVED_PROJECTION_ALIAS = "TEST_OTHER_DERIVED_PROJECTION_ALIAS";
 
     private static final String TEST_OTHER_DERIVED_PROJECTION_EXPRESSION = "TEST_OTHER_DERIVED_PROJECTION_EXPRESSION";
 
-    private RouteUnit routeUnit = mock(RouteUnit.class);
+    private RouteUnit routeUnit;
 
     @Before
     public void setup() {
         RouteMapper routeMapper = mock(RouteMapper.class);
         when(routeMapper.getLogicName()).thenReturn(TEST_LOGIC_TABLE_NAME);
-        when(routeMapper.getActualName()).thenReturn(TEST_ACTUAL_TABLE_NAME);
+        final String testActualTableName = "TEST_ACTUAL_TABLE_NAME";
+        when(routeMapper.getActualName()).thenReturn(testActualTableName);
```

Comment:
If the variable is only used once, it can be used directly.
Refactored to ‘ when(routeMapper.getActualName()).thenReturn("TEST_ACTUAL_TABLE_NAME")’.

Reply:
Okay, I will fix it. 
If the variable is a int instead of a string, should I also use it directly?